src/pages/iou/IOUModal.js

```diff
@@ -458,12 +458,18 @@ class IOUModal extends Component {
                                             {this.renderHeader()}
                                             <IOUConfirmPage
                                                 onConfirm={(selectedParticipants) => {
```

Comment:
Can we have the comment below this line here? I think it looks a bit noisy to have it repeated in both functions.